ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py

```diff
@@ -141,3 +141,31 @@ def test_numpy_isfortran(
         on_device=on_device,
         a=x[0],
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.iscomplexobj",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("real_and_complex"), min_num_dims=1
+    ),
+    test_with_out=st.just(False),
+)
+def test_numpy_iscomplexobj(
+    dtype_and_x,
+    frontend,
+    on_device,
+    *,
+    fn_tree,
+    test_flags,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=x[0],
+    )
+
+
```

Comment:
Same here: `ivy_tests/test_ivy/test_frontends/test_numpy/test_logic/test_truth_value_testing.py:171:1: W391 blank line at end of file`

Reply:
Hi ZoeCD i will install pre-commit and try to fix the lint errors,
thanks for highlighting.